ext/ParameterSchedulersFluxExt.jl

```diff
@@ -0,0 +1,55 @@
+
+module ParameterSchedulersFluxExt
+
+using ParameterSchedulers, Flux
+
+export Scheduler
```

Comment:
```suggestion
```
Is this export required/does it do anything? Otherwise this LGTM, based on the commit history I'm guessing you've verified the docstrings work as expected :)

Reply:
Scheduler is not exported in main. So we can remove this line.